pkg/dockerutil/dockerutils.go

```diff
@@ -1982,6 +1973,9 @@ func CanRunWithoutDocker() bool {
 	if output.ParseBoolFlag("version", "v") || output.ParseBoolFlag("help", "h") {
 		return true
 	}
+	if len(os.Args) == 2 && output.ParseBoolFlag("json-output", "j") {
+		return true
+	}
```

Comment:
I noticed that `DOCKER_HOST=foo ddev -j` doesn't work without Docker, but it should.